subtract.py

```diff
@@ -0,0 +1,3 @@
+c=10
+d=5
```

Comment:
why is the code hardcoded?

Reply:
This is just a sample code, will modify to dynamic values by getting input from user.

Reply:
Then do it now

Reply:
Ok I will update now in this PR

Reply:
I have fixed the comments